### ISSUE TITLE
hydra:title is overwriting in DocumentationNormalizer

### ISSUE THREAD

**Issue by xoxmaman** [api-platform/core#1294]
vendor/api-platform/core/src/Hydra/Serializer/DocumentationNormalizer.php:245

```php
elseif ('POST' === $method) {
            $hydraOperation = [
                '@type' => 'hydra:CreateResourceOperation',
                'hydra:title' => "Creates a $shortName resource.",
                'returns' => $prefixedShortName,
                'expects' => $prefixedShortName,
            ] + $hydraOperation;
}
```

There must be a reverse

```php
elseif ('POST' === $method) {
            $hydraOperation = $hydraOperation + [
                '@type' => 'hydra:CreateResourceOperation',
                'hydra:title' => "Creates a $shortName resource.",
                'returns' => $prefixedShortName,
                'expects' => $prefixedShortName,
            ];
} 
```

**Comment by meyerbaptiste** [comment]
Huum, that could make sense if we want to override the default operation... I don't know if there's a reason for the current behavior.

WDYT @api-platform/core-team?

**Comment by Simperfit** [comment]
I agree that makes sense, IMO it's a bug.

**Comment by soyuka** [comment]
I'm fixing this.

**Comment by Simperfit** [comment]
Fixed by #1295
